cogs/Lifecycle.py

```diff
@@ -623,6 +628,10 @@ async def compileveto(self, ctx: commands.Context):
                 await vetoDiscussionChannel.send(
                     content=mysteryHellMessage[i : i + hc_constants.LITERALLY_1984]
                 )
+                copyableText += mysteryHellMessage + "\n\n"
+        await vetoDiscussionChannel.send(
+            content=f"Copy and paste below this line:\n{copyableText}"
+        )
```

Comment:
mork dies if it tries to send a message of 2k characters and i think this runs into it, see also: why 1984 is here